src/hooks/use-check-light-curate.ts

```diff
@@ -1,37 +1,18 @@
-import { useEffect, useState } from 'react'
-import { useWeb3Context } from 'web3-react'
-import { ethers } from 'ethers'
+import { useMemo } from 'react'
 import { useParams } from 'react-router'
-import _gtcr from '../assets/abis/GeneralizedTCR.json'
+import { useQuery } from '@apollo/client'
+import { TCR_EXISTENCE_TEST } from 'utils/graphql'
 
-const useCheckLightCurate = () => {
-  const { library, active } = useWeb3Context()
+const useCheckLightCurate = (): [boolean, boolean] => {
```

Comment:
returning arrays is confusing, would be more clear if it returned instead:
`{ isLightCurate: boolean, loading: boolean } `